web-api/src/main/java/com/graphhopper/jackson/MultiException.java

```diff
@@ -26,6 +26,7 @@ public class MultiException extends RuntimeException {
     private final List<Throwable> errors;
 
     public MultiException(List<Throwable> errors) {
+        super(errors.toString());
```

Comment:
Since we did not set the `message` field for the super exception we got this ugly `MultiException: null` in our logs.